app/(tabs)/account.tsx

```diff
@@ -6,6 +6,9 @@ const AccountScreen = () => {
 
   return (
     <View style={styles.container}>
+      <Text style={styles.email}>Logged in as: {user?.id}</Text>
+      <Text style={styles.email}>Logged in as: {user?.accessToken}</Text>
```

Comment:
Do we need to display the access token here?

Reply:
Just temporary until I do something with that page, mostly it's for testing auth and storage. The whole component will be redesigned in the near future.